Source/WebCore/rendering/TextPaintStyle.h

```diff
@@ -48,6 +48,7 @@ struct TextPaintStyle {
     Color strokeColor;
     Color emphasisMarkColor;
     float strokeWidth { 0 };
+    bool hasExplicitlySetFillColor { false };
```

Comment:
This name is fine because of the use of `fillColor` here, but we should use a more appropriate name in `RenderStyle`.

Reply:
In `StyleRareInheritedData.h` there is a `hasSetStrokeColor` so I followed that name. What name do you think would be more appropriate?

Reply:
`stroke-color` is the name of the CSS property. In this case, the property we're checking is `color`.

`setHasExplicitlySetColor` / `hasExplicitlySetColor` in the `RenderStyle` adjacent code would be more appropriate.

Though as mentioned above, keeping it as `hasExplicitlySetFillColor` in `TextPaintStyle` is fine, since we refer to the "fill color" there.